Altis_Life.Altis/core/functions/fn_loadGear.sqf

```diff
@@ -1,87 +1,44 @@
 #include "..\..\script_macros.hpp"
 /*
     File: fn_loadGear.sqf
-    Author: Bryan "Tonic" Boardwine
+    Author: Bryan "Tonic" Boardwine
+    Edited: blackfisch
```

Comment:
I've tried to discourage this on recent PRs as if everyone does this, it slowly will become a little out of hand - it was an intention to remove them. Using it for a little change log would be better than just writing Edited though!

Reply:
sounds logic to me :)